todoapp/todos/serializers.py

```
@@ -1,5 +1,34 @@
 from django.contrib.auth import get_user_model
 from rest_framework import serializers
 
+from todos import models as todos_models
+from users import serializers as users_serializers
+
 
 # Add your serializer(s) here
+class TodoSerializer(serializers.ModelSerializer):
+    creator = users_serializers.UserSerializer(
+        fields=["first_name", "last_name", "email"], source="user"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)